jobs/models.py

```diff
@@ -89,13 +89,18 @@ class Meetup(models.Model):
     description = models.TextField()
     is_recurring = models.BooleanField(
         default=False,
-        help_text="Is your meetup recurring?"
     )
     #TODO this field should be required if the is_recurring is True
-    recurrence = models.CharField(max_length=255, blank=True, null=True)
+    recurrence = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        help_text="Provide reccurence if your meetup is reccuring."
+    )
     meetup_date = models.DateTimeField(
-        null = True,
-        help_text="This stands for a starting date if the meetup is recurring"
+        null=True,
+        help_text="This stands for a starting date if the meetup is recurring.\
+            Date format: YYYY-MM-DD"
```

Comment:
We should be able to set a date format in the form; let's use a more global standard if we can (numerical date formats are ambiguous)